gateway/api-gateway-config.json

```diff
@@ -2,7 +2,7 @@
   "sso": {
     "plan": "uaa"
   },
-  "count": 2,
+  "count": 1,
```

Comment:
Since we don't have to use Dhaka env for testing, we can continue with 2 instances of gateway which may help surfacing some multi-instance issues.